src/Shout-Tests/SHRBTextStylerTest.class.st

```diff
@@ -31,96 +31,13 @@ SHRBTextStylerTest >> style: aText [
 	^ ast
 ]
 
-{ #category : #'private - utilities' }
-SHRBTextStylerTest >> styleExpression: aText [
-
-	| ast |
-
-	ast := self class compiler
-		source: aText asString;
-		noPattern: true ;
-		options:  #(+ optionParseErrors + optionSkipSemanticWarnings);
-		parse.
-	styler style: aText ast: ast.
-
-	^ ast
-]
-
 { #category : #running }
 SHRBTextStylerTest >> tearDown [
 
 	SHRBTextStyler formatIncompleteIdentifiers: oldSetting.
 	super tearDown
 ]
 
-{ #category : #tests }
```

Comment:
Where are the new tests equivalent to those removed?

Reply:
Yes, but not relying on the current theme! Some of those were in reality failing but we didn’t see it since it was relying on the theme

Reply:
ah, sorry, I did not see the `load diff` hiding a lot of changes.